addons/l10n_dk/data/account_tax_template_data.xml

```diff
@@ -422,125 +600,168 @@
             }),
             (0,0, {
-                'factor_percent': -100,
+                'factor_percent': -92.83,
                 'repartition_type': 'tax',
-                'account_id': ref('a8769'),
             }),
         ]"/>
-        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+    </record>
+    <!-- 0.896 cent per kilowatt (to set as quantity) can be deduced -->
+    <!-- https://skat.dk/data.aspx?oid=2246453 -->
+    <record id="tax720" model="account.tax.template">
+        <field name="chart_template_id" ref="dk_chart_template"/>
+        <field name="sequence">720</field>
+        <field name="name">Elafgift</field>
+        <field name="description"></field>
+        <field name="amount">0.896</field>
+        <field name="amount_type">fixed</field>
+        <field name="type_tax_use">purchase</field>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'base',
             }),
             (0,0, {
-                'factor_percent': -25,
+                'factor_percent': 100,
                 'repartition_type': 'tax',
-                'account_id': ref('a8730'),
+                'account_id': ref('a8775'),
+                'plus_report_line_ids': [ref('l10n_dk.account_tax_report_line_deduction_electrical_tax')],
             }),
             (0,0, {
-                'factor_percent': 25,
+                'factor_percent': -100,
                 'repartition_type': 'tax',
-                'account_id': ref('a8740'),
+            }),
+        ]"/>
+        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'base',
             }),
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'tax',
-                'account_id': ref('a8750'),
+                'account_id': ref('a8775'),
+                'minus_report_line_ids': [ref('l10n_dk.account_tax_report_line_deduction_electrical_tax')],
             }),
             (0,0, {
                 'factor_percent': -100,
                 'repartition_type': 'tax',
-                'account_id': ref('a8769'),
             }),
         ]"/>
     </record>
-
-    <record id="tax520" model="account.tax.template">
+    <record id="tax730" model="account.tax.template">
         <field name="chart_template_id" ref="dk_chart_template"/>
-        <field name="sequence">520</field>
-        <field name="name">EU Ydelseskøb</field>
+        <field name="sequence">730</field>
+        <field name="name">Naturgas- og bygasafgift</field>
         <field name="description"></field>
-        <field name="price_include" eval="0"/>
-        <field name="amount">100</field>
-        <field name="amount_type">percent</field>
+        <field name="active" eval="False"/>
+        <field name="amount">4.4490</field>
```

Comment:
Is this per liter or something?  It does not make any sense to create a fixed tax with an amount that is clearly not fixed even with active False.  What was wrong with the 100% idea?

Reply:
Several things here:
1) The 100 percent that it deletes doesn't concern this tax: git just failed to matched the lines because there is a lot of line added.

2) The 4.4490 amount was set by our partner, regarding this, here is a discussion I had with Kenneth: https://github.com/odoo/odoo/pull/80823#discussion_r761773625
The number is an excise tax and isn't VAT nor can't be deduced from VAT. (at least that's what I learned in school regarding the general EU principles of VAT)

3) Thinking about it, and re-reading the energy deduction page from the skat, I think the system is the same than other energy tax: 
- it can't be easily calculated but is mentioned on the invoice
- has a deduction lower than 100% of 92.83% that can vary over time

Considering (3), I will just give it the same implementation than the `Olie- og flaskegasafgift` tax.